tck/microprofile-tck/pom.xml

```diff
@@ -63,4 +63,134 @@
       </modules>
     </profile>
   </profiles>
+
+  <dependencyManagement>
```

Comment:
Any reason for adding them under `dependencyManagement` in the `microprofile-tck`  again? The only difference is the `scope = test`, which is decleared on each artifact in the related sub-modules?

Reply:
i believe this is not a duplicate, there are artifacts for API and more artifacts for TCK tests.